server/routes/questions/router.test.ts

```diff
@@ -328,6 +331,24 @@ describe(`Submitting questions' responses`, () => {
       },
     ]
 
+    const questionsResponse: Question[] = [
+      {
+        code: '45054',
+        question: 'WERE THE POLICE INFORMED OF THE INCIDENT',
+        additionalInformation: null,
+        responses: [
+          {
+            response: 'YES',
+            responseDate: new Date(),
```

Comment:
technically `responseDate` is `parseDateInput(responseDate)`

Reply:
This is to mock the return value of `addOrUpdateQuestionsWithResponses()` which returns `Question[]` (well, a `Promise<Question[]` to be precise). So responseDate is actually a `Date`, unless I'm missing something 🤔

Reply:
yes it's a date, but technically not “now” ;)